baggage/templates/baggage_list (còpia).html

```diff
@@ -0,0 +1,60 @@
+{% extends "base_table.html" %}
+
+{% load static %}
```

Comment:
Isn't this from Baggage?

Reply:
Yes, removed, somehow the folder was there...